### ISSUE TITLE
Bring back a prerelease indicator in version

### ISSUE THREAD

**Issue by luser** [mozilla/rr#1440]
While implementing https://github.com/mozilla/rr/pull/1437 I wished that the version number would differentiate between a prerelease and official release. I saw that @cgjones had removed this at some point in the past and there's an exasperated comment in the source, so maybe this is hard to do?
https://github.com/mozilla/rr/blob/0e9fe8332674f4ce36e0ae079c4ba4c57884a7f6/CMakeLists.txt#L12

**Comment by cgjones** [comment]
Nah, we just haven't kept up with it.  Here's the scheme we used to use:

https://github.com/mozilla/rr/blob/d8845e87b6d7a2d9506654388a34b5a7777b125e/CMakeLists.txt#L12
